content/general/concepts/progressive-web-application/progressive-web-application.md

```diff
@@ -0,0 +1,36 @@
+---
+Title: 'Progressive Web Application'
+Description: 'A progressive web application provides a consistent, browser-based experience across multiple platforms like mobile and desktop.'
+Subjects:
+  - 'Mobile Development'
+  - 'Web Development'
+Tags:
+  - 'Accessibility'
+  - 'Browsers'
+  - 'Browser Compatibility'
+  - 'Development'
+CatalogContent:
+  - 'introduction-to-javascript'
+  - 'paths/front-end-engineer-career-path'
+---
+
+<link rel="canonical" href="https://www.codecademy.com/resources/blog/what-is-a-progressive-web-application/" />
+
+A progressive web application (PWA) is a software that is built with common web technologies to provide a consistent, browser-based experience across multiple platforms like mobile and desktop. Browser-compatbility for PWAs is various but is generally supported in the following:
+
+- Google Chrome
+- Apple Safari
+- Firefox (for Android, not desktop)
+- Microsoft Edge
+
+## PWA Principles
+
+Rather than being a programming language or separate technology, PWAs are a collection of key principles followed in every application, including:
+
+- Discoverability: The app can be readily found on app stores and search engines.
+- Installability: The app can be installed as a "native-like app" on a device for later use on the browser.
+- Linkability: A unique link can be used to visit and share the app.
+- Network independence: The app use provides an offline experience with service workers and other web APIs.
+- Re-engageability: The app can update content when not in use and push notifications.
```

Comment:
I think "Re-engagement" makes sense because the app is updating in the background put pushes notifications to the user so that when they revisit, there is new content.

What are your thoughts?